### PR TITLE
Feature: Add paybutton name, uuid and buttonData fields — Change paybuttonsService

### PR DESCRIPTION
**Pull request by chedieck**
Description:
Part of #33: Makes the necessary changes related to the new added fields to  `paybuttonsService.ts` and its tests.

Depends on:

- [x] #113 
- [x] #114 

Test plan:
The API & tests will be addressed in another PR, so those tests will fail. `yarn docker t tests/unittests/paybuttonsService.test.ts` should succeed.

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -1,39 +1,46 @@
...
-  createdAt           DateTime              @default(now())
-  updatedAt           DateTime              @updatedAt
-  paybuttonAddresses  PaybuttonAddress[]
-
+  id                 Int                @id @default(autoincrement())
+  slug               String             @unique @db.VarChar(255)
+  title              String             @db.VarChar(255)
+  createdAt          DateTime           @default(now())
+  updatedAt          DateTime           @updatedAt
+  paybuttonAddresses PaybuttonAddress[]
```
Comment (Klakurka) [inline prisma/schema.prisma:45]: I feel like I already reviews this stuff in #114

Comment (chedieck) [reply prisma/schema.prisma:45]: Yes, this PR depends on #114, it includes all code from there.

Comment (Klakurka) [reply prisma/schema.prisma:45]: Ok I'll look at this again after it's merged into master.

**Review by Klakurka — CHANGES_REQUESTED** [review]
Rebase with master.

**Comment by chedieck** [comment]
> Rebase with master.

Done.

**Review by Klakurka — DISMISSED** [review]

File: services/paybuttonsService.ts [hunk 2]
```diff
@@ -10,9 +10,9 @@ export interface CreatePaybuttonInput {
   prefixedAddressList: string[]
 }
 
-export async function createPaybutton (userId: string, prefixedAddressList: string[]): Promise<Paybutton> {
+export async function createPaybutton (values: CreatePaybuttonInput): Promise<Paybutton> {
```
Comment (Gattaca1) [inline services/paybuttonsService.ts:13]: ~~The API wasn't updated to use this new format, but no test catches that.

- Add a test that will fail (does the paybutton actually get created?)
- Update the API to use this new `createPayButton` argument format

EDIT: There is another test that fails in the integration test. After I fixed that, the integration test now fails to create the paybutton, so ignore the previous test feedback. Just update the API~~

I see that all the API stuff is added in a separate commit. This looks good to go then

**Review by Klakurka — APPROVED** [review]

**Review by Gattaca1 — APPROVED** [review]

**Pull request merged**